torch/fx/experimental/fx2trt/fx2trt.py

```diff
@@ -287,6 +287,7 @@ def run_node(self, n):
         return super().run_node(n)
 
     def placeholder(self, target, args, kwargs):
+        assert self.optimization_profiles
```

Comment:
We probably want to add it below line 299 because `self.optimization_profiles` could be None in some of the cases.